pkg/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -178,6 +178,9 @@ var _ = Describe("Tuned", func() {
 				})
 				It("should set by interface name with reserved CPUs count", func() {
 					netDeviceName := "eth*"
+					//regex field should be: devices_udev_regex=r'^INTERFACE_NAME=eth.*''
+					devicesUdevRegex := "r'\\^INTERFACE_NAME=" + strings.Replace(netDeviceName, "*", "\\.\\*", -1) + "'"
```

Comment:
I meant to make this a string literal but no big deal, not worth a respin just for this.